backend/github/auth.py

```
@@ -6,89 +6,93 @@
+
+        url = f"https://api.github.com/app/installations/{self.installation_id}/access_tokens"
         headers = {
-            'Authorization': f'Bearer {jwt_token}',
-            'Accept': 'application/vnd.github.v3+json',
+            "Authorization": f"Bearer {jwt_token}",
+            "Accept": "application/vnd.github.v3+json",
         }
-        
+
         try:
             response = requests.post(url, headers=headers)
```

Ruff (0.15.20)

[error] Probable use of `requests` call without timeout

(S113)